### ISSUE TITLE
Build Feature 1.13+

### ISSUE THREAD

**Issue by FrostyZz** [cabaletta/baritone#852]
I am happy to announce i have started work on the new block ID's! At the end of this project the #build feature will now be usable in versions 1.13+. If you would like any updates, have any questions, or want to help the process (I'd love some help xD) Add me on Discord at Frosty#9190 and MSG me what your contacting about so i know who you are. @leijurv if you end up seeing this, id love to talk to you as-well.

**Comment by leijurv** [comment]
its simple you just need to make something that can load the new spongepowered schematic file format thingy

i just dont care enough to do it since i dont play above 1.12.2 lol

**Comment by FrostyZz** [comment]
yeah, if good with you id like to chat with you over discord if you have it (just assuming you do) to grab some info and to grab your help if needed. (its simpler that coming to github all the time. @leijurv

**Comment by leijurv** [comment]
#5021

**Comment by FrostyZz** [comment]
ok @leijurv i added u and sent u a msg telling that it was me :) hope to talk with you soon.

**Comment by dayangvvv** [comment]
> its simple you just need to make something that can load the new spongepowered schematic file format thingy
> 
> i just dont care enough to do it since i dont play above 1.12.2 lol

i really really really need that in 1.13.2 
would you plz update it?
